tests/fixtures/logs.py

```diff
@@ -24,3 +24,21 @@ def gelf_logger():
     logger.addHandler(handler)
 
     return logger
+
+
+def event_factory(factory_class, size=1, **kwargs):
+    """Returns `size` number of events using the `factory_class`.
+
+    Args:
+        factory_class (Factory): The factory to use to generate events
+        size (integer): Number of events to generate
+        kwargs: Pass additional keyword arguments to the factory
+
+    Returns:
+        A dict representing the event when `size` is 1
+        A `size` long list of events when `size` is more than 1
+    """
+
+    if size == 1:
+        return factory_class.create(**kwargs).dict()
+    return [event.dict() for event in factory_class.create_batch(size, **kwargs)]
```

Comment:
What do you think of always returning a generator instead?

Reply:
Good idea) Thanks) As this method would become a single line, I have removed it)